generate_zeebe_initial_contact.py

```diff
@@ -0,0 +1,28 @@
+""""This script generates the initial contact string for Zeebe clusters in a multi-region setup."""
+
+# For now the script is focused on dual-region.
+# It will be extended if anything specific changes for more than 2 regions.
+
+def generate_string(ns_0, ns_1, release, count):
+    """Generates the initial contact string for Zeebe clusters in a multi-region setup."""
+    result = ""
+    for i in range(count // 2):
+        result += f"{release}-zeebe-{i}.{release}-zeebe.{ns_0}.svc.cluster.local:26502,"
```

Comment:
A small suggestion to move `26502` into an inner constant (`port_number` or similar) to avoid a magic constant reference.